index.html

```diff
@@ -0,0 +1,143 @@
+              <img src="./assets/images/png/mail (image).png" alt="Logo" />
+            </div>
+            <div class="logo__text">NinjaMail</div>
+          </div>
+
+          <div class="footer__nav">
+            <div class="footer__column">
+              <ul>
+                <li class="footer__link">
+                  <a href="">Features</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Pricing</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Services</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Partners</a>
+                </li>
+              </ul>
+            </div>
+            <div class="footer__column">
+              <ul>
+                <li class="footer__link">
+                  <a href="">About Us</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Tutorials</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Resources</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Help Center</a>
+                </li>
+                <li class="footer__link">
+                  <a href=""> Templates</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Case Studies</a>
+                </li>
+              </ul>
+            </div>
+            <div class="footer__column">
+              <ul>
+                <li class="footer__link">
+                  <a href=""> Medium</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Twitter</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Facebook</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Instagram</a>
+                </li>
+                <li class="footer__link">
+                  <a href=""> Linked In</a>
+                </li>
+              </ul>
+            </div>
+            <div class="footer__column">
+              <ul>
+                <li class="footer__link">
+                  <a href="">Contact Us</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Slack</a>
+                </li>
+                <li class="footer__link">
+                  <a href="">Jobs</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+        <div class="footer__bottom">
+          <div class="footer__item">
+            NinjaMail is a sample project for Quest AI. © 2019 Quest AI, All
+            rights reserved.
+          </div>
+          <div class="footer__item">Terms & Conditions</div>
+          <div class="footer__item">Privacy Policy</div>
+        </div>
+      </footer>
+    </div>
```

Comment:
in many cases you don't have to add a DIV tag for all tags. You can simply divide according to the meaning of the sections. For example:
body
|
header
|
section
|
section
|
footer
|
body